readme/welcome/4_tips.md

```diff
@@ -4,7 +4,7 @@ The first few notes should have given you an overview of the main functionalitie
 
 ## Web clipper
 
-![](./WebClipper.png)
+![Image of the WebClipper](./WebClipper.png)
```

Comment:
What's the recommendations for alt text in general? It's seems redundant to say that an image is an image of something? And what does "image of the webclipper" is going to tell to a visually impaired person - they'll still have no idea what it looks like? Aren't we supposed to describe the content of the image?

Reply:
The Web Accessibility Initiative's  [ALT text decision tree might be helpful](https://www.w3.org/WAI/tutorials/images/decision-tree/). Based on this, there are several cases where the description doesn't need to be in the ALT text:
- If it doesn't contribute meaning to the page, use an empty ALT.
- If it shows content "redundant to real text nearby", use an empty ALT.
- If it's a more complicated piece of information (e.g. a graph), describe the content elsewhere on the page.
- ...other cases...

If, based on the decision tree, this image should be using an empty "alt" attribute, something to check might be that the Markdown maps `![](:/someidhere)` to an `img` with `alt=""` (which might be appropriate in some cases, but not in many others).

Reply:
"Image of" is redundant, but I think this one could simply read "screenshot of WebClipper browser pop-up".

Reply:
To be honest I'm not seeing any value in any of the alt texts that were added. We have a paragraph about the mobile and desktop applications, then we add an image with alt text "image of mobile and desktop applications". That gives almost no information to someone who's visually impaired. Most of these are probably under category "redundant to real text nearby"